lib/peak_tracker_web/controllers/peak_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule PeakTrackerWeb.PeakController do
+  use PeakTrackerWeb, :controller
+
+  alias PeakTracker.Mountains
+
+  def index(conn, _params) do
+    peaks = Mountains.Peak
+    |> Ash.Query.sort([:name])
```

Comment:
Is it okay to put an `Ash.Query` in a controller or should it rather be put in the `Mountains` module in this case?